src/tests/end-to-end/tests/details-view/__snapshots__/tabstop.test.ts.snap

```diff
@@ -0,0 +1,60 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Tabstop tests tabstop from fastpass if visualHelper is turned on after starting tabstop and pressing tabs on target page 1`] = `
+<DocumentFragment>
+  <div
+    id="insights-shadow-container"
+  >
+    <link
+      href="{{EXTENSION_URL_BASE}}/injected/styles/default/injected.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="{{EXTENSION_URL_BASE}}/bundle/injected.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <div
+      class="insights-container insights-highlight-container insights-issues"
```

Comment:
This looks off.... by enabling tab stops visualization I can see this on the target page: 
```html
<div class="insights-container insights-highlight-container insights-tab-stops">
```
but the snapshot doesn't have this div... it has one with `insights-issues` for class name which shows up when enabling automated checks... there is something off here for sure.

Reply:
Updated.